lametro/bills.py

```diff
@@ -94,7 +104,7 @@ def actions(self, matter_id) :
 
                 yield bill_action, votes
 
-    def scrape(self, window=28, matter_ids=None) :
+    def scrape(self, window=28, matter_ids=None, scrape_restricted=True) :
```

Comment:
let's just reuse the `self.scrape_restricted` property?

Reply:
oh, yeah! that was my original plan and I somehow changed course.